app/Services/SearchService.php

```diff
@@ -261,6 +261,16 @@ public function find()
             return $this->newOptions();
         }
 
+        foreach ($searchResults as $result) {
```

Comment:
We can save resources by only doing an if first that checks that it's not New, to return the results directly, rather than a foreach